sdk/purview/purview-workflow-rest/samples-dev/workflowRunSample.ts

```diff
@@ -0,0 +1,116 @@
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to List workflow runs.
+ *
+ * @summary List workflow runs.
+ * x-ms-original-file: specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/examples/ListWorkflowRuns.json
+ */
+async function workflowRunsList(
+  client: PurviewWorkflowClient,
+  queryParameter: ListWorkflowRunsParameters
+) {
+  const initialResponse = await client.path("/workflowruns").get(queryParameter);
+
+  if (isUnexpected(initialResponse)) {
+    throw initialResponse.body.error;
+  }
+  const pageData = paginate(client, initialResponse);
+  const result = [];
+  for await (const item of pageData) {
+    result.push(item);
+  }
+  console.log(
+    `The total count of workflow runs is ${result.length}, these workflow runs are ${result}`
+  );
+}
+
+/**
+ * This sample demonstrates how to Get a workflow run.
+ *
+ * @summary Get a workflow run.
+ * x-ms-original-file: specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/examples/GetWorkflowRun.json
+ */
+async function workflowRunGet(client: PurviewWorkflowClient, workflowRunId: string) {
+  const result = await client.path("/workflowruns/{workflowRunId}", workflowRunId).get();
+  if (isUnexpected(result)) {
+    throw result.body.error;
+  }
+  console.log(`The returned workflow run is ${result.body}`);
+}
+
+/**
+ * This sample demonstrates how to Cancel a workflow run.
+ *
+ * @summary Cancel a workflow run.
+ * x-ms-original-file: specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/examples/CancelWorkflowRun.json
+ */
+async function workflowRunCancel(
+  client: PurviewWorkflowClient,
+  workflowRunId: string,
+  cancelReply: CancelWorkflowRunParameters
+) {
+  const result = await client
+    .path("/workflowruns/{workflowRunId}/cancel", workflowRunId)
+    .post(cancelReply);
+  if (isUnexpected(result)) {
+    throw result.body.error;
+  }
+  console.log(`Cancel workflow run ${workflowRunId} successfully.`);
+}
+
+async function main() {
+  // ================================================== Create client ==================================================
+
+  const endpoint = process.env["ENDPOINT"] || "";
+  const tenantId = process.env["TENANTID"] || "";
+  const clientId = process.env["CLIENTID"] || "";
+  const username = process.env["USERNAME"] || "";
+  const password = process.env["PASSWORD"] || "";
+  const credential = new UsernamePasswordCredential(tenantId, clientId, username, password);
+  const client = createPurviewWorkflowClient(endpoint, credential);
+
+  // ================================================== List workflow runs ==================================================
+
+  const queryParameters: ListWorkflowRunsParameters = {
+    queryParameters: {
+      timeWindow: "30d",
+      orderby: "startTime desc",
+      maxpagesize: 1000,
+    },
+  };
+  workflowRunsList(client, queryParameters);
```

Comment:
Same here, lets do this inline. It is easier to follow the flow of the sample

Reply:
The same before, I think keep main() simple will help user understand the flow more easier.